cell_classification/segmentation_data_prep.py

```diff
@@ -62,33 +77,43 @@ def get_image(self, data_folder, marker):
             data_folder (str):
                 The path to the data_folder
             marker (str):
-                The marker shown in the image, e.g. "CD8" corresponds to
-                file name "CD8.tiff" in the data_folder
+                The marker shown in the image, e.g. "CD8" corresponds
+                to file name "CD8.tiff" in the data_folder
+        Returns:
+            np.array:
+                The multiplexed image
         """
-        pass
+        return np.zeros([500, 500])
 
-    def make_binary_mask(self, instance_mask):
+    def get_instance_masks(self, data_folder, cell_mask_key):
         """Makes a binary mask from an instance mask by eroding it
 
         Args:
-            instance_mask (np.array):
-                The instance mask to make binary
+            data_folder (str):
+                The path to the data_folder
+            cell_mask_key (str):
+                The key in the data_folder that contains the cell mask labels
         Returns:
             np.array:
                 The binary mask
+            np.array:
+                The instance mask
         """
-        pass
+        return np.zeros([500, 500]), np.zeros([500, 500])
 
-    def get_cell_types(self, labels):
+    def get_cell_types(self, labels, sample_name):
```

Comment:
The arguments here don't match up with the arguments down below in the example. 

I don't think we need the labeled image though, can't we just subset the `cell_table` to the name of image we're looking at to get the list of cells present in the image? Is there a reason to use the mask?

Reply:
Yes, it should  be enough to subset the `cell_table` with the name of the image. I included the labels, because I thought that the images will be tiled before the next steps. But now I am thinking it'll be best if tiling is done at last in function `tile_example`.